### ISSUE TITLE
Add MAKEFILE for port on other platforms

### ISSUE THREAD

**Issue by Viladevita** [xuancong84/ProjectDIVA#1]
please, we need porperly work Switch port, i have some issue withreverse engineering youre project

**Comment by xuancong84** [comment]
I don't think it is easy to port to other platform due to Microsoft proprietary DirectX technology  -:(

**Comment by DheikoGW** [comment]
at using other data the game wont open

**Comment by DheikoGW** [comment]
![image](https://user-images.githubusercontent.com/71303198/151646722-210a34da-3c13-4f32-a3d8-de1c88f1c753.png)
![image](https://user-images.githubusercontent.com/71303198/151646725-3c53dbc7-94b4-48b0-b4b0-4cf96d8f7897.png)
At using other data the game wont open, Please, How i Fix this?

**Comment by xjunko** [comment]
> ![image](https://user-images.githubusercontent.com/71303198/151646722-210a34da-3c13-4f32-a3d8-de1c88f1c753.png) ![image](https://user-images.githubusercontent.com/71303198/151646725-3c53dbc7-94b4-48b0-b4b0-4cf96d8f7897.png) At using other data the game wont open, Please, How i Fix this?

Might be necro but whatever, you can fix this by downloading `msvcp100d.dll` and `msvcr100d.dll` somewhere online (the top results usually works), the guy who maintained this repo probably forgot to build a release version, so it's using the debug dlls.

**Comment by DheikoGW** [comment]
Thanks man!!!!!

Em dom., 9 de out. de 2022 às 04:18, Junko ***@***.***>
escreveu:
